swagger.json

```diff
@@ -5101,15 +5101,41 @@
                 "label",
                 "value"
             ],
+            "description": "A field of a document. A field can be a string type, number type, or photo type. Between stringValue, numberValue, and photoValue, only one can exist for a single document field depending on the field's type.",
             "properties": {
                 "label": {
                     "type": "string",
                     "description": "Descriptive name of this field.",
                     "example": "Fuel Cost ($)"
                 },
                 "value": {
-                    "description":  "Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
+                    "description":  "DEPRECATED: Please use stringValue, numberValue, or photoValue instead. Value of this field. Depending on what kind of field it is, this may be one of the following: an array of image urls, a float, an integer, or a string.",
                     "example": 23.45
+                },
+                "stringValue": {
+                  "type": "string",
+                  "description": "Value of this field if this document field is a string type field.",
+                   "example": "This is a string."
+                },
+                "numberValue": {
+                  "type": "number",
+                  "format": "float",
+                  "description": "Value of this field if this document field is a number type field.",
+                   "example": 12.34
```

Comment:
ooc: do we plan on providing POST endpoints for documents in the future? If so, will the `Document` definition be required to somehow support validation? (eg - number of decimal places)

The current implementation seems good for GET endpoints, but this might be something to think about so we don't end up having to duplicate definitions or deprecate `numberValue` prematurely.

Reply:
Yes. Once we hit the graphql endpoint, that will handle validations for num decimal places. Also, presumably they should fetch the document template to see how many decimal places they should use for this value.

Reply:
Nvm, talked in person. Seems like we don't plan on re-using this def for document template POST, so the lack of validation shouldn't be an issue.